### PR TITLE
Add custom config option to linux installer script

### PR DESCRIPTION
**Pull request by jeffreyc-splunk**

### CODE REVIEW

**Review by rmfitzpatrick — APPROVED** [review]

**Comment by jeffreyc-splunk** [comment]
> You may want to document this in https://github.com/signalfx/splunk-otel-collector/blob/main/docs/getting-started/linux-installer.md#collector-configuration ?

Good point, thanks!

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+### 💡 Enhancements 💡
+
+- Add `--collector-config` option to the Linux installer script to allow a custom config file path (#1806)
```
Comment (aurbiztondo-splunk) [inline CHANGELOG.md:7]: Add the `--collector-config` option...

File: docs/getting-started/linux-installer.md [hunk 2]
```diff
@@ -109,6 +109,17 @@ modified as needed. Possible configuration options can be found in the
 - [OpenTelemetry Collector](https://github.com/open-telemetry/opentelemetry-collector)
 - [OpenTelemetry Collector Contrib](https://github.com/open-telemetry/opentelemetry-collector-contrib)
 
+To use an existing collector configuration file instead of the default, run
```
Comment (aurbiztondo-splunk) [inline docs/getting-started/linux-installer.md:112]: I'd write: 

the installer script with the `--collector-config PATH_TO_CONFIG` option, replacing `PATH_TO_CONFIG` with the absolute path to the desired configuration file on the host. For example:

**Review by aurbiztondo-splunk — APPROVED** [review]
Left a couple of minor comments re. wording.

**Pull request merged**